Fix flex/bison checks (allow full path to be given)

--- configure.ac~	2007-08-27 03:44:20.000000000 +0200
+++ configure.ac	2018-11-20 12:11:25.115828450 +0100
@@ -30,14 +30,14 @@
 AC_PROG_CC
 AC_PROG_AWK
 AC_PROG_LEX
-if test "$LEX" != "flex"; then
+if test "${LEX##*/}" != "flex"; then
 	AC_MSG_ERROR([flex not found])
 fi
 if test -z "`echo %%|$LEX -t|grep yypop_buffer_state`"; then
 	AC_MSG_ERROR([flex missing yypop_buffer_state - upgrade to version 2.5.33 or later])
 fi
 AC_PROG_YACC
-if test "$YACC" != "bison -y"; then
+if test "${YACC##*/}" != "bison -y"; then
 	AC_MSG_ERROR([bison not found])
 fi
 
